docs/UserGuide.adoc

```diff
@@ -152,7 +154,7 @@ Format: `sort|<key>`
 
 ****
 * Sorting by both name and date is done in ascending order.
```

Comment:
Perhaps change to "Sorting by both name (lexicographical order) and date (chronological order)". Also see if we can reduce the number of times we use chronological/lexicographical order. Maybe 2 times or less is fine. 3 or more could sound quite repeatative.

Reply:
Yeps, actually I was wondering if we could just state somewhere in the beginning of Section 3 that the lists displayed for addcommands etc. will be in lexicographical and chronological order so that in the other sections we don't have to explicitly state that again?

Reply:
Noted. Will make the changes.

Reply:
How about just mentioning it once in the `view` section?